fixtures/webhook_call/find.yaml

```diff
@@ -0,0 +1,72 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: [identity]
+      Authorization: [Bearer foobar]
+      Connection: [keep-alive]
+      Content-Type: [application/vnd.contentful.management.v1+json]
+      User-Agent: [python-requests/2.18.1]
+      X-Contentful-User-Agent: [sdk contentful-management.py/1.3.1; os macOS/16.5.0;
+          platform python/3.5.2;]
+    method: GET
+    uri: https://api.contentful.com/spaces/orzkxlxlq59d/webhooks/16ypL3XjNK6oreLPPoVBxI/calls/4KYbXQ9cg8CE2aqouWqY2i
+  response:
+    body: {string: "{\n  \"sys\": {\n    \"id\": \"4KYbXQ9cg8CE2aqouWqY2i\",\n    \"type\":
```

Comment:
So we could migrate all gzipped bodies to identities, for the sake of readability?